frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/SlideInDialog.kt

```diff
@@ -0,0 +1,61 @@
+package com.virtuslab.gitmachete.frontend.actions.dialogs
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.DialogWrapper
+import com.intellij.openapi.ui.ValidationInfo
+import com.intellij.ui.DocumentAdapter
+import com.intellij.ui.layout.CellBuilder
+import com.intellij.ui.layout.ValidationInfoBuilder
+import com.intellij.ui.layout.panel
+import com.virtuslab.gitmachete.frontend.actions.common.GitMacheteBundle.message
+import kotlin.apply
+import kotlin.text.isEmpty
+import kotlin.text.trim
+
+class SlideInDialog constructor(project: Project,
```

Comment:
Nit: merge lines?

Reply:
I thought about it in the first place but splitting constructor args seems to be a convention. 
I see two reasons to keep it like it is now:
- the other Kotlin constructors I saw within IntelliJ code have the split params
- (probably the key reason) the argument preceded with `var` or `val` becomes a field and splitting them increases readability

Kotlin note:
`val` - constant 
`var` - regular variable

Reply:
whoops, I could not force kotlin formatter to use "my" approach. finally lines merged

Reply:
Seems they're not merged now ;)